apps/react/src/features/anime/components/AnimePageParams/AnimePageParams.tsx

```diff
@@ -0,0 +1,91 @@
+import { useQueryParams } from '@js-camp/react/hooks/use-query-params';
+import { Box, FormControl, InputLabel, MenuItem, Select, TextField } from '@mui/material';
+import { FC, memo } from 'react';
+import { AnimeType } from '@js-camp/core/models/anime-type';
+import { selectAreAnimeLoading } from '@js-camp/react/store/anime/selectors';
+import { useAppSelector } from '@js-camp/react/store';
+import { AnimeFieldEnum } from '@js-camp/core/models/anime';
+import { splitNonEmptyString } from '@js-camp/react/utils';
+import { extractValidAnimeFilters } from '@js-camp/react/features/utils';
+import { QueryParamKeys } from '@js-camp/react/models/query-params';
+
+const englishTitle = AnimeFieldEnum.EnglishTitle;
+const status = AnimeFieldEnum.Status;
+
+const SEARCH_DEBOUNCE_TIME = 500;
+
+/**
+ * Creates sort value.
+ * @param field Sort field.
+ * @param direction Sort direction.
+ */
+function createSortValue(field: AnimeFieldEnum, direction: '' | '-'): string {
+	return `${direction}${field}`;
+}
+
+const sorts = [
```

Comment:
Let's improve type safety here. Also in case you have multiple select components in your application, this type could be put somewhere more globally for reusability.
```suggestion
type SelectOption = Readonly<{

  /** Label. */
  label: string;

  /** Value. */
  value: string;
}>;

const sortOptions: SelectOption[] = [
```